internal/controller/vrg_volrep.go

```diff
@@ -2016,6 +2017,15 @@ func (v *VRGInstance) restorePVsAndPVCsFromS3(result *ctrl.Result) (int, error)
 			continue
 		}
 
+		// If in failover mode and no PVs are found, try next S3 profile
+		if v.instance.Spec.Action == ramendrv1alpha1.VRGActionFailover && pvCount == 0 {
```

Comment:
The issue is not only failover.  It is for failover and relocation. I would just change it like this:
```
if v.instance.Spec.Action != "" && pvCount == 0 {
    v.log.Info("No PVs found in the s3 store", "S3ProfileName", s3ProfileName)
    
    continue
}
```